bin-ranges-transfer/src/main/java/uk/gov/pay/java_lambdas/bin_ranges_transfer/util/Utils.java

```diff
@@ -16,6 +17,8 @@
 public class Utils {
 
     private static final Logger logger = LoggerFactory.getLogger(Utils.class);
+    private static final Pattern datePattern = Pattern.compile("\\d{8}");
+    private static final Pattern versionPattern = Pattern.compile("V\\d{2}");
```

Comment:
Should really be `SCREAMING_SNAKE_CASE`.

Reply:
done